packages/server/src/modules/chat/resolvers.js

```diff
@@ -0,0 +1,144 @@
+import fs from 'fs';
+import mkdirp from 'mkdirp';
+import shell from 'shelljs';
+import { createBatchResolver } from 'graphql-resolve-batch';
+
+const MESSAGE_SUBSCRIPTION = 'message_subscription';
+const MESSAGES_SUBSCRIPTION = 'messages_subscription';
+const UPLOAD_DIR = 'public';
+
+const storeFS = ({ stream, filename }) => {
```

Comment:
Looks great, but It's the same func as we are using in the Upload module. As sentence, we can move this functionality in separate file or module in the future.